services/src/main/java/org/jboss/windup/web/services/rest/FileEndpoint.java

```diff
@@ -18,4 +18,12 @@
     @POST
     @Path("pathExists")
     boolean pathExists(String path);
+
+    @POST
+    @Path("pathTargetType")
+    PathTargetType pathTargetType(String path);
```

Comment:
I don't think this is really used anywhere, but since it is here, I will review it. It lacks Javadoc and the formatting of the implementation is wrong.